cosmovisor/go.mod

```diff
@@ -1,131 +1,126 @@
 module github.com/cosmos/cosmos-sdk/cosmovisor
 
-go 1.17
+go 1.18
 
 require (
-	github.com/cosmos/cosmos-sdk v0.45.1
+	github.com/cosmos/cosmos-sdk v0.46.0-alpha3
```

Comment:
still haven't had time to read up on workspaces, can this be left to 0.45.x?

Reply:
..... this is a maybe